projects/packages/my-jetpack/src/class-rest-product-data.php

```diff
@@ -107,14 +107,14 @@ public static function get_site_backup_undo_event() {
 			}
 		}
 
-		return rest_ensure_response( $undo_event, 200 );
+		return rest_ensure_response( $undo_event );
 	}
 
 	/**
 	 * This will collect a count of all the items that could be backed up
 	 * This is used to show what backup could be doing if it is not enabled
 	 *
-	 * @return array
+	 * @return WP_Error|\WP_REST_Response
```

Comment:
Usually, these kind of changes to php docs "irritate" phan 😅 maybe the static check will pass if you revert it / change to `WP_Error|array`?

Reply:
Hmm this was throwing a warning since `array` is not what is returned 🤔 But it's been this way for a while and if it's going to cause some phan shenanigans then maybe I'll just revert that

Reply:
It seems like that didn't fix the phan errors 😓 Maybe I'll just move it back to array and see if that helps

Reply:
I reverted it back to array and I'm still getting the PHAN errors 😢

Reply:
I saw your conversation in jetpack devs channel, glad you got it fixed!